mayastor/src/bdev/nexus/nexus_module.rs

```diff
@@ -145,6 +151,41 @@ impl NexusModule {
     extern "C" fn nexus_ctx_size() -> i32 {
         std::mem::size_of::<NioCtx>() as i32
     }
+
+    /// creates a JSON object that can be applied to mayastor that
+    /// will construct the nexus object and its children. Note that
+    /// the nexus implicitly tries to create the children as such
+    /// you should not have any iSCSI create related calls that
+    /// construct children in the config file.
+    extern "C" fn config_json(w: *mut spdk_json_write_ctx) -> i32 {
+        instances().iter().for_each(|nexus| {
+            let uris = nexus
+                .children
+                .iter()
+                .map(|c| c.name.clone())
+                .collect::<Vec<String>>();
+
+            let json = json!({
+                "method": "create_nexus",
+                "params": {
+                    "uuid" : nexus.name.as_str()[6 ..],
```

Comment:
Maybe should explain the 6 chars here "nexus-" ?. Could it ever change? 
I thought "crypto-" can get prefixed onto the name as well?

Reply:
Good question.  But the one level "up" so the crypto instance is stacked ontop of it.